### PR TITLE
mantis shrimp facts

### PR DESCRIPTION
**Pull request by dancwpark**

### CODE REVIEW

File: animalfacts.py [hunk 1]
```diff
@@ -2249,6 +2263,7 @@ def animalfactsbot(reddit):
     LIZARD_FACTS,
     LOBSTER_FACTS,
     LLAMA_FACTS,
```
Comment (PratBudhiraja) [inline animalfacts.py]: LLAMA_FACTS = (
'The llama is a South American relative of the camel, though the llama does not have a hump.',
'These sturdy creatures are domestic animals used by the peoples of the Andes Mountains.',

File: animalfacts.py [hunk 2]
```diff
@@ -1306,6 +1307,19 @@ def animalfactsbot(reddit):
...
+    'A Mantis Shrimp can punch at a speed of 10 meters per second, equivalent to a .22 bullet.',
+    'Mantis Shrimp are known to break glass aquariums with their punches.',
+    'Mantis Shrimp are not actually shrimp, they are instead stomatopods, a distant relative.',
+    'A punch of a Mantis Shrimp momentarily causes the temperature of the surrounding water to reach the temperature of the Sun.',
+    'Mantis Shrimp use their ability to see polarized light to commmunicate with other mantis shrimp in a way that is invisible to predators.',
+    'Mantis Shrimp can make a low growling sounds and often grunt at dawn and dusk.',
+    'Stomatopods, like Mantis Shrimp, are older than dinosaurs.'
+    )
+
 MEERKAT_FACTS = (
```
Comment (ArinnaBee) [inline animalfacts.py]: 'Meerkats are omnivores with a diet consisting mostly of insects, though they also eat small rodents, fruit, birds, eggs, lizards, and even poisonous scorpions.',

File: animalfacts.py [hunk 3]
```diff
@@ -2249,6 +2263,7 @@ def animalfactsbot(reddit):
     LIZARD_FACTS,
     LOBSTER_FACTS,
```
Comment (victototoria) [inline animalfacts.py]: Meerkats are social creatures as they live in groups called mobs, that can contain up to 40 members.

File: animalfacts.py [hunk 4]
```diff
@@ -1306,6 +1307,19 @@ def animalfactsbot(reddit):
...
+    'Mantis Shrimp have trinocular vision, meaning they can see using three parts of the same eye.',
+    'Mantis Shrimp can be categorized into "Spearers" and "Smashers", referencing tactics used to kill their prey.',
+    'A Mantis Shrimp can punch at a speed of 10 meters per second, equivalent to a .22 bullet.',
+    'Mantis Shrimp are known to break glass aquariums with their punches.',
+    'Mantis Shrimp are not actually shrimp, they are instead stomatopods, a distant relative.',
+    'A punch of a Mantis Shrimp momentarily causes the temperature of the surrounding water to reach the temperature of the Sun.',
+    'Mantis Shrimp use their ability to see polarized light to commmunicate with other mantis shrimp in a way that is invisible to predators.',
+    'Mantis Shrimp can make a low growling sounds and often grunt at dawn and dusk.',
+    'Stomatopods, like Mantis Shrimp, are older than dinosaurs.'
+    )
```
Comment (AcidThunder) [inline animalfacts.py]: Mantis Shrimp grow to an average of 10cm length

Comment (AcidThunder) [inline animalfacts.py]: 'Mantis Shrimp grow to an average 10cm length' #222

File: animalfacts.py (hunk 2 above)
Comment (werpish2) [inline animalfacts.py]: 'Meerkats, or suricates, are a type of mongoose that live in the southern African plains, primarily the countries of Namibia, South Africa, and Botswana.',

File: animalfacts.py [hunk 5]
```diff
@@ -1306,6 +1307,19 @@ def animalfactsbot(reddit):
...
+
```
Comment (chatz2) [inline animalfacts.py]: Mantis shrimps have the most complex eyes in the animal kingdom that can see ultraviolet and polarized light. They have trinocular vision which means that they can see one objects using one of the three different parts of eye.

File: README.md [hunk 6]
```diff
@@ -93,6 +93,7 @@ You can see the bot in action at https://www.reddit.com/user/AnimalFactsBot/comm
 * lizard
 * lobster
 * llama
```
Comment (devestations) [inline README.md]: Llamas are vegetarians and have efficient digestive systems.

**Pull request merged**